<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDomain</class>
 <widget class="QWidget" name="TimeDomain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>771</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>激发脉冲参数设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>脉冲频率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseFrequency">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseNum">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>脉冲个数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HZ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePulseInterval">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>脉冲间隔：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>平均次数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAverageTimes">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>采样点数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSampleNum">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>HZ</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spStartPosition">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spEndPosition">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>us</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnReadPulsePara">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnWritePulsePara">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>写入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>延迟时间：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spDelayTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>测量操作</string>
   </property>
   <widget class="QToolButton" name="tbnFrequencyDomainNext">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>频域</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnSaveData">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnStartDraw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnClearWaves">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnChangeWaveStyle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnLoadData">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbnReset">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>起始位置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>终止位置：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTimeDomainMinValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>最大值：：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>最小值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>峰峰值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTimeDomainMaxValue">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTimeDomainMaxMinValue">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxComPort">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="leDotNum">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="leDotText">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>画点个数：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>1121</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>时域波形展示</string>
   </property>
   <widget class="QCustomPlot" name="MyCustomPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1091</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
